apps/src/code-studio/header.js

```diff
@@ -351,6 +366,24 @@ header.showMinimalProjectHeader = function() {
   $('.project_remix').click(remixProject);
 };
 
+// Levelbuilder-only UI for saving changes to a level.
+header.showLevelBuilderSaveButton = function(getChanges) {
+  var projectName = $('<div class="project_name_wrapper header_text">')
```

Comment:
This header UI is still in jQuery 😢. I opened https://codedotorg.atlassian.net/browse/LP-131 to track.